doc/src/region.rst

```diff
@@ -210,9 +210,11 @@ equal-style :doc:`variable <variable>`.  Likewise, for style *sphere*
 and *ellipsoid* the x-, y-, and z- coordinates of the center of the
 sphere/ellipsoid can be specified as an equal-style variable.  And for
 style *cylinder* the two center positions c1 and c2 for the location of
-the cylinder axes can be specified as a equal-style variable.  For style
-*cone* all properties can be defined via equal-style variables.  For
-style *plane* the point can be defined via equal-style variables.
+the cylinder axes can be specified as a equal-style variable.  For styles
+*cone* and *plane* all properties can be defined via equal-style variables.
+For style *plane*, there is the restriction that all three components
+of the direction normal to the plane should be either constant or
+defined via equal-style variables.
 
 If the value is a variable, it should be specified as v_name, where
```

Comment:
Somewhere on the region doc page I think you should add a Note (which will appear in a blue box, see the HTML version of the region doc page).  Which cautions users whenever they use a variable to define a region property, to ensure that the variable does not cause any part of the region boundary to move too far within a single timestep. "Too far" would be more than a small fraction of a particle "size".  Otherwise bad dynamics can result.  The Note could also say this is true of the move/rotate options.  It could also highlight region styles for which this is easy to mess up.  E.g. region plane normals (this PR), since a small change in normal could shift the region surface far away from the region point by a large displacement.

Reply:
@sjplimp what is a particle "size"? The simulations are usually carried out for point particles (such as atoms) which have no radius - of course granular materials are an obvious exception.

Reply:
I was using "size" generically to mean the approx distance of closest approach for 2 particles, but it would be good to clarify that.  So sigma for LJ particles, or diameter for granular particles, etc.  Small fraction of that distance should ensure there is no large jump in the wall force in a single timestep,